examples/react-native/src/features/Authenticator/Styles/Example.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ReactNode } from 'react';
+import { StyleProp, StyleSheet, Text, View, ViewStyle } from 'react-native';
+
+import { Authenticator, useAuthenticator } from '@aws-amplify/ui-react-native';
+import { Amplify } from 'aws-amplify';
+
+import { Button } from '../../../ui';
+
+Amplify.configure({});
+
+const MyHeader = ({
+  children,
+  style,
+}: {
+  children?: ReactNode;
+  style?: StyleProp<ViewStyle>;
+}) => (
+  <View style={style}>
+    <Text style={{ fontSize: 36 }}>{children}</Text>
+  </View>
+);
+
+function SignOutButton() {
+  const { signOut } = useAuthenticator();
+  return <Button onPress={signOut}>Sign Out</Button>;
+}
+
+function App() {
```

Comment:
Just noting this example is not complete and will be used for the `style` prop handling example